<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WeatherManager</class>
 <widget class="QWidget" name="WeatherManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>820</width>
    <height>585</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WeatherManager</string>
  </property>
  <widget class="QLabel" name="tempLabel">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>120</y>
     <width>111</width>
     <height>121</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string>temp</string>
   </property>
  </widget>
  <widget class="QPushButton" name="themeButton">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>30</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>ChangeTheme</string>
   </property>
  </widget>
  <widget class="QPushButton" name="voiceButton">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>100</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Voice</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="forecastTable">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>290</y>
     <width>671</width>
     <height>261</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>日期</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>天气</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>温度</string>
    </property>
   </column>
  </widget>
  <widget class="QLineEdit" name="cityLineEdit">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>60</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="searchButton">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>20</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cityComboBox">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>10</y>
     <width>67</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="weatherIcon">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>120</y>
     <width>111</width>
     <height>121</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Icon</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
